lib/bap_llvm/llvm_elf_loader.hpp

```diff
@@ -204,28 +204,29 @@ void symbol_entry(const ELFObjectFile<T> &obj, const SymbolRef &sym, ogre_doc &s
     }
 }
 
-#if LLVM_VERSION_MAJOR == 3 && LLVM_VERSION_MINOR == 8
+#if LLVM_VERSION_MAJOR == 3 && LLVM_VERSION_MINOR == 8          \
+    || LLVM_VERSION_MAJOR == 4 && LLVM_VERSION_MINOR == 0
```

Comment:
Per LLVM devs the MINOR version will always be 0. Its only the patch version that will be updated during the release intermezzo. I think that we can safely ignore the patch version though.